example/declarative.dart

```diff
@@ -1,29 +1,31 @@
 // Copyright 2013 Google. All rights reserved. Use of this source code is
 // governed by a BSD-style license that can be found in the LICENSE file.
 
-library grinder_example_1;
+library grinder.example.declarative;
 
 import 'package:grinder/grinder.dart';
 
-void main([List<String> args]) {
-  task('init', init);
-  task('compile', compile, ['init']);
-  task('deploy', deploy, ['compile']);
-  task('default', null, ['deploy']);
-
+void main(List<String> args) {
+  addAnnotatedTasks();
   startGrinder(args);
 }
 
-void init(GrinderContext context) {
+@Task()
```

Comment:
It would be nice to be able to tighten this up, to something like `@task`. Not sure how possible that is given there's already a function named `task`.